rules/aip0126/unspecified_test.go

```diff
@@ -29,9 +29,9 @@ func TestUnspecified(t *testing.T) {
 	}{
 		{"Valid", "BookFormat", "BOOK_FORMAT_UNSPECIFIED", testutils.Problems{}},
 		{"ValidWithNum", "Ipv6Format", "IPV6_FORMAT_UNSPECIFIED", nil},
+		{"ValidUnknown", "BookFormat", "UNKNOWN", nil},
 		{"InvalidNoPrefix", "BookFormat", "UNSPECIFIED", testutils.Problems{{Suggestion: "BOOK_FORMAT_UNSPECIFIED"}}},
 		{"InvalidWrongSuffix", "BookFormat", "BOOK_FORMAT_UNKNOWN", testutils.Problems{{Suggestion: "BOOK_FORMAT_UNSPECIFIED"}}},
-		{"InvalidJustWrong", "BookFormat", "UNKNOWN", testutils.Problems{{Suggestion: "BOOK_FORMAT_UNSPECIFIED"}}},
```

Comment:
Funny that this was the "JustWrong" test case, but is actually an exception in the AIP. Must've fallen out of sync.